models/bronze/adventure_works.yml

```
@@ -4,6 +4,12 @@ sources:
   - name: adventure_works
     database: FEA24_11
     schema: RAW_ADVENTURE_WORKS
+
+    freshness:
+      warn_after: {count: 12, period: hour}
+      error_after: {count: 24, period: day}

```

```suggestion
      error_after: {count: 100000, period: day}

```
Mudança para não dar erro nos teste de freshness